test/io/ferguson/recordeof.good

```diff
@@ -0,0 +1 @@
+Done
```

Comment:
Comments in the file did not make it clear that the test was expected to read EOF immediately (and succeed).  Did I miss the comment that addresses this?